### ISSUE TITLE
IMDb integration

### ISSUE THREAD

**Issue by Light7654** [LagradOst/CloudStream-3#563]
Please Add IMDb tracking

**Comment by Bungeexgum** [comment]
> Please Add IMDb tracking

Nah, they can't! This app is pathetic only provide Anime/Movie, no recommendations/related like Kuro does. I mean how can we know the if there's sequel/prequel? Only synopsis is there nothing else. The only good about this app are the player/subtitle and many features in setting.

**Comment by ITADORI-15** [comment]
What is kuro used for?
Can u provide link if it is for movies or series(mainly)

**Comment by Bungeexgum** [comment]
> What is kuro used for? Can u provide link if it is for movies or series(mainly)

It is for Anime Only, if you are looking for Movie App try these two https://www.showbox.media https://hdo.app     Available for Android only.

**Comment by ITADORI-15** [comment]
Can u also send kuro's link.....for alternative....plzzz

**Comment by Light7654** [comment]
> Can u also send kuro's link.....for alternative....plzzz
> 

https://github.com/deceptions/no

**Comment by Bungeexgum** [comment]
> Can u also send kuro's link.....for alternative....plzzz

Use this app instead It's the best Anime App right now https://github.com/saikou-app/saikou/releases      

But if you wanna check Kuro here it is https://github.com/deceptions/no

**Comment by ITADORI-15** [comment]
Thnx

**Comment by Buddie911** [comment]
Hi guys, is sflix working for you guys in the app???

**Comment by Light7654** [comment]
> Hi guys, is sflix working for you guys in the app???

Yeah but 1080p is missing

**Comment by Buddie911** [comment]
Really? In the latest pre or? Which version are you using? Im on the latest pre and all i get is no links found

**Comment by ITADORI-15** [comment]
Same

**Comment by Buddie911** [comment]
Same no links found right??? Or its working for you?

**Comment by ITADORI-15** [comment]
It isnt working

**Comment by Buddie911** [comment]
Yup i thought so,on  their website first two servers aren't working either so maybe thats the problem

**Comment by ITADORI-15** [comment]
Just c......from the topic IMDb integration....we hve come so far

**Comment by Buddie911** [comment]
Lmao right

**Comment by LagradOst** [comment]
> > Please Add IMDb tracking
> 
> Nah, they can't! This app is pathetic only provide Anime/Movie, no recommendations/related like Kuro does. I mean how can we know the if there's sequel/prequel? Only synopsis is there nothing else. The only good about this app are the player/subtitle and many features in setting.

ye fuck you too

**Comment by Raif1** [comment]
> > > Please Add IMDb tracking
> > 
> > Nah, they can't! This app is pathetic only provide Anime/Movie, no recommendations/related like Kuro does. I mean how can we know the if there's sequel/prequel? Only synopsis is there nothing else. The only good about this app are the player/subtitle and many features in setting.
> 
> ye fuck you too

Do you plan to do so ? Like adding imdb and mal rating in the series movies anime?
